src/editor/Command.cpp

```diff
@@ -117,6 +117,9 @@ void DeleteEntityCommand::execute() {
 	map->ents.erase(map->ents.begin() + entIdx);
 
 	refresh();
+
+	map->update_ent_lump();
+	map->update_lump_pointers();

```

Comment:
I think I see the problem now with the entity importing. These method calls should be in the export code instead. Otherwise it would need to be copy-paste this code to all entity-related commands and controls.

Reply:
Yes i see. Fixed. And commands reverted back.